group-coordinator/src/main/java/org/apache/kafka/coordinator/group/assignor/SimpleAssignor.java

```diff
@@ -56,155 +58,80 @@
  * Balance is prioritized above stickiness.
  */
 public class SimpleAssignor implements ShareGroupPartitionAssignor {
-
+    private static final Logger LOG = LoggerFactory.getLogger(SimpleAssignor.class);
```

Comment:
nit: almost all files in Kafka uses `log`.
```suggestion
    private static final Logger log = LoggerFactory.getLogger(SimpleAssignor.class);
```